WebwinkelKeur/registration.php

```diff
@@ -0,0 +1,23 @@
+<?php
+namespace WebwinkelKeur\Magento2;
+
+use Magento\Framework\Component\ComponentRegistrar;
+
+ComponentRegistrar::register(
+    ComponentRegistrar::MODULE,
+    'WebwinkelKeur_Magento2',
+    isset($file) && realpath($file) == __FILE__ && getenv('VALUED_DEVELOPMENT') == 'yes'
+        ? dirname($file) : __DIR__
+);
+
+spl_autoload_register(function ($cls) {
+    $prefix = __NAMESPACE__ . '\\Controller\\Adminhtml\\';
```

Comment:
This solution works, but I am in doubt about it. The provided[ code](https://gist.github.com/apeschar/4c9a6e603bb188dfb6cfd9e49680833d) from you should do the job, however, 
when visiting **TrustProfile\Magento2\Controller\Adminhtml\Actions\Import**  `$cls` is equal to **TrustProfile\Magento2\Controller\Adminhtml\Noroute**.
In the current situation, it works because there is nothing else under **TrustProfile\Magento2\Controller\Adminhtml**, but this is hardcoded, and in case another file is added to Adminhtml this has to be changed.
Should I go for extending `Import` from `Valued` namespace?